content/en/docs/setup/install/external-controlplane/test.sh

```diff
@@ -104,10 +104,14 @@ snip_set_up_the_control_plane_in_the_external_cluster_2
 snip_get_external_istiod_iop_modified
 echo y | istioctl install -f external-istiod.yaml --context="${CTX_EXTERNAL_CLUSTER}" --set values.pilot.env.ISTIOD_CUSTOM_HOST="${EXTERNAL_ISTIOD_ADDR}"
 
-_verify_like snip_set_up_the_control_plane_in_the_external_cluster_5 "$snip_set_up_the_control_plane_in_the_external_cluster_5_out"
+_verify_like snip_set_up_the_control_plane_in_the_external_cluster_6 "$snip_set_up_the_control_plane_in_the_external_cluster_6_out"
+
+snip_set_up_the_control_plane_in_the_external_cluster_6
+
+snip_set_up_the_control_plane_in_the_external_cluster_5
```

Comment:
Is there an issue in the docs/tests if the tests is running snippet 5 after snippet 5?